docs/en/operations/system-tables/dead_letter_queue.md

```diff
@@ -0,0 +1,71 @@
+---
+slug: /en/operations/system-tables/dead_letter_queue
+---
+# dead_letter_queue
+
+Contains information about messages received via a stream engine and parsed with an errors. Currently implemented for Kafka and RabbitMQ.
```

Comment:
Maybe `streaming engine` is also okay.
```suggestion
Contains information about messages received via a stream like engine and parsed with an errors. Currently implemented for Kafka and RabbitMQ.
```

Reply:
Thanks. I think I like "streaming engine" more.